2026-02-27 Hopp (#318)
rustacean-station/rustacean-station.org: _episodes/interviews/2026-02-27-hopp.md

---
title: "Hopp with Costa Alexoglou and Iason Paraskevopoulos"

> date: 2026-02-27T09:37:00Z

Reviewer: ```suggestion
date: 2026-02-27T11:25:00Z
```

file: https://audio.rustacean-station.org/file/rustacean-station/2026-02-27-hopp.mp3
duration: "42:43"
length: "102522240"
#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
---

> Allen Wyma talks with [Costa Alexoglou](https://blog.alexoglou.com/) and [Iason Paraskevopoulos](https://iparaskev.com/), creators of [Hopp](https://www.gethopp.app/), a tool that is made for developers to have a seamless remote pairing session. Hopp is built with Tauri and Rust and is open source.

Reviewer 2: ```suggestion
Allen Wyma talks with [Costa Alexoglou](https://blog.alexoglou.com/) and [Iason Paraskevopoulos](https://iparaskev.com/), creators of [Hopp](https://www.gethopp.app/), a tool that is made for developers to have a seamless remote pair programming session. Hopp is built with Tauri and Rust and is open source.
```

> Allen Wyma talks with [Costa Alexoglou](https://blog.alexoglou.com/) and [Iason Paraskevopoulos](https://iparaskev.com/), creators of [Hopp](https://www.gethopp.app/), a tool that is made for developers to have a seamless remote pair programming session. Hopp is built with Tauri and Rust and is open source.

Reviewer: ```suggestion
Allen Wyma talks with [Costa Alexoglou](https://blog.alexoglou.com/) and [Iason Paraskevopoulos](https://iparaskev.com/), creators of [Hopp](https://www.gethopp.app/), a tool that enables remote pair programming sessions. Hopp is built with Tauri and Rust, and is open source.
```
